src/components/CircularProgressbar.js

```diff
@@ -1,17 +1,24 @@
-import { CircularProgressbar } from 'react-circular-progressbar';
+import { CircularProgressbar, buildStyles } from 'react-circular-progressbar';
 import 'react-circular-progressbar/dist/styles.css';
 
 const CircularProgress = () => {
-  const percentage = 70;
+  const percentage = 64;
+  const alpha = percentage / 100;
+
   return (
     <div className="percentage">
-      <CircularProgressbar value={percentage} />
+      <CircularProgressbar
+        value={percentage}
+        styles={buildStyles({
+          pathColor: `rgba(2, 144, 255, ${alpha})`,
+        })}
+      />
```

Comment:
**[OPTIONAL]**

- [ ] It would be nice if you make your own logo of your name and put that in the readme file at the top to present yourself as a brand in the job market or tech industry to catch the recruiters' eyes 👍

Reply:
After the suggestion i create a logo and add it to the Read me